src/qonnx/__init__.py

```diff
@@ -1,5 +1,13 @@
-import onnxruntime
+from qonnx import converters  # noqa: F401
 
 
 def reseed(newseed):
+    import numpy
+    import onnxruntime
+    import tensorflow
+
+    print(f"pytest-randomly: reseed with {newseed}")
+
     onnxruntime.set_seed(newseed)
+    tensorflow.random.set_seed(newseed)
+    numpy.random.seed(seed=newseed)
```

Comment:
Several comments on this `reseed` function:
1. it seems to be unused and if so should be removed
2. pytest-randomly is mentioned but is not installed (and if so, it should be made an optional testing dependency)
3. this function does not belong in the top-level init, the utilities module or a conftest fixture like https://stackoverflow.com/a/68222757/1520469 may be a better place

Reply:
Hi @maltanar, I installed `pytest-randomly` in a previous PR: https://github.com/fastmachinelearning/qonnx/blob/main/setup.cfg#L74

`reseed` is used as part of the test suite (called impliclty by `pytest-randomly`). We can place it wherever we want as long as we correctly configure it here: 

https://github.com/fastmachinelearning/qonnx/blob/main/setup.cfg#L86-L87